app/src/main/res/menu/episodes.xml

```diff
@@ -17,6 +17,14 @@
         custom:showAsAction="always"
         android:icon="@drawable/ic_refresh"/>
 
+    <item
+        android:id="@+id/add_podcast_item"
+        android:title="@string/add_feed_label"
+        android:menuCategory="container"
+        custom:showAsAction="always"
+        android:visible="false"
+        android:icon="@drawable/ic_add"/>
```

Comment:
This feels pretty hacky. It is not really a context menu entry of an episode.

Reply:
Hmm yea, I think we could do without this, it's kind of a relic from PowerEpisodes :D